sdk/communication/AzureCommunicationChat/Tests/ChatClientTests.swift

```diff
@@ -221,4 +234,80 @@ class ChatClientTests: XCTestCase {
             }
         }
     }
+
+    func test_StartPushNotifications_ReturnsSuccess() {
+        let expectation = self.expectation(description: "Start push notifications")
+
+        chatClient.startPushNotifications(deviceToken: "mockDeviceToken") {
+            result in
+            switch result {
+            case let .success(response):
+                XCTAssertEqual(response?.statusCode, 202)
+            case .failure:
+                XCTFail("Start push notifications failed.")
+            }
+
+            expectation.fulfill()
+        }
+
+        waitForExpectations(timeout: 10000.0) { error in
+            if let error = error {
+                XCTFail("Start push notifications timed out: \(error)")
+            }
+        }
+    }
+
+    func test_StopPushNotifications_ReturnsSuccess() {
+        let expectation = self.expectation(description: "Stop push notifications")
+
+        // Start notifications first
+        chatClient.startPushNotifications(deviceToken: "mockDeviceToken") {
+            result in
+            switch result {
+            case .success:
+                // Stop notifications
+                self.chatClient.stopPushNotifications { result in
+                    switch result {
+                    case let .success(response):
+                        XCTAssertEqual(response?.statusCode, 202)
+                    case .failure:
+                        XCTFail("Stop push notifications failed.")
+                    }
+
+                    expectation.fulfill()
+                }
+            case .failure:
+                XCTFail("Start push notifications failed.")
+                expectation.fulfill()
+            }
+        }
+
+        waitForExpectations(timeout: 100.0) { error in
+            if let error = error {
+                XCTFail("Stop push notifications timed out: \(error)")
+            }
+        }
+    }
+
+    func test_StopPushNotifications_ReturnsFailure() {
+        let expectation = self.expectation(description: "Stop push notifications")
+
+        // Stop notifications without starting them
+        chatClient.stopPushNotifications { result in
+            switch result {
+            case .success:
+                XCTFail("Push notifications should not be enabled.")
+            case let .failure(error):
+                XCTAssertNotNil(error)
+            }
+
+            expectation.fulfill()
+        }
+
+        waitForExpectations(timeout: 100.0) { error in
+            if let error = error {
+                XCTFail("Stop push notifications timed out: \(error)")
+            }
+        }
+    }
```

Comment:
These are all XCRecordableTests in ChatClientDVRTests and should not be duplicated here.